src/lib/rule-finder.js

```diff
@@ -57,20 +61,19 @@ function _getPluginRules(config) {
 
       const rules = pluginConfig.rules === undefined ? {} : pluginConfig.rules;
       pluginRules = pluginRules.concat(
-        Object.keys(rules).map(rule => {
-          return normalized.prefix + '/' + rule;
-        })
+        Object.keys(rules)
+        .filter(rule => !_isDeprecated(rules[rule]))
```

Comment:
Yes, good catch.  I think I was following the pattern of something else in the file.  I'll fix that up.